src/main/java/me/leoko/advancedban/manager/UpdateManager.java

```diff
@@ -59,6 +59,7 @@ public void setup() {
             addMessage("UnBan:", "  Notification: \"&e&o%OPERATOR% &7unbanned &c&o%NAME%\"", 1);
             addMessage("UnMute:", "  Notification: \"&e&o%OPERATOR% &7unmuted &c&o%NAME%\"", 1);
             addMessage("UnWarn:", "  Notification: \"&e&o%OPERATOR% &7unwarned &c&o%NAME%\"", 1);
+            addMessage("UnNote:", "  Notification: \"&e&o%OPERATOR% &7unnoted &c&o%NAME%\"", 1);
```

Comment:
This is not how the UpdateManager works. Please take a look at it again and adjust your code. Feel free to ask if you need any help understanding how it works.

Reply:
That was 100% my bad. Misunderstood what the class was. And didn't think twice because I reset the configs several times during my tests